### ISSUE TITLE
New User Request

### ISSUE THREAD

**Issue by fujah06** [ArjunaCluster/ArjunaUsers#187]
### Andrew ID

oajenifu

### Group Name

Amir Barati Farimani

### Are you subscribed to arjuna-users@lists.andrew.cmu.edu?

Yes, or I have requested access to the list serve

### I will not block emails from root@coe.psc.edu

Yes

### I have read and will abide by the Acceptable Use Policy.

Yes

**Comment by fujah06** [comment]
Hello ArjunaCluster/ArjunaUsers,
I made a request for an access to arjuna cluster, and I got a message
stating "closed as requested " I dont know what it means. I will appreciate
if you could clarify this for me.


Thanks,
Olabode Ajenifujah.

**Comment by fujah06** [comment]
I am trying to install OpenFOAM software on Arjuna, and the installing involved using sudo and sourcing in bashrc, I have the instruction in the link attached below 
https://www.cemf.ir/how-to-install-openfoam-v2012-from-source-pack-on-ubuntu/

I have been able to carry step 1-3, but I could not carry out step 4 and the rest because it requires sudo and sourcing bashrc. I will appreciate if you could guide or give access so that I can install the software. I currently have it on my local computer, but I need it on Arjuna.


Thanks,
Olabode.

**Comment by awadell1** [comment]
Please open a new issue for different issues.

See our docs for info about installing software on Arjuna: https://arjunacluster.github.io/ArjunaUsers/about/user_software.html

Tldr; we do not provide sudo access to users; additional software beyond what’s provided via modules must be installed via spack/from source.
